sourcespec/config_files/configspec.conf

```diff
@@ -318,9 +318,18 @@ geom_spread_cutoff_distance = float(min=0, default=100)
 
 
 # INVERSION PARAMETERS --------
-# Weighting type: 'noise', 'frequency' or 'no_weight'
-weighting = option('noise', 'frequency', 'no_weight', default='noise')
-# Parameters for 'frequency' weighting (ignored for 'noise' weighting):
+# Weighting type: 'noise', 'frequency', 'inv_frequency' or 'no_weight'
+#   'noise':         spectral signal/noise ratio weighting
+#   'frequency':     a constant weight is applied for f<=f_weight
+#                    a weight of 1 is used for f>f_weight
+#                    (see "f_weight" and "weight" below)
+#   'inv_frequency': weight is computed as 1/(f-f0+0.25)**0.25 for f<=f1,
+#                    weight is 0 for f<f0 and f>f1.
+#                    f0 and f1 are the first and last frequencies where
+#                    spectral signal/noise ratio is above 3
```

Comment:
@krisvanneste, do you agree with the above explanation?

Reply:
Yes, maybe add:
- `(if noise window is available)`
or
- `or the first and last frequencies of the entire spectrum if no noise window is available`